ptf-tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ptf
+paramiko
+scapy==2.4.5
+pyang==2.6.0

```

Comment:
Same comment as other PR.
pyang version used in ptf-tests repo is 2.5.2. Would this break with an untested version?

Reply:
I generally accept Dependabot's recommendations.

Pyang 2.5.2 was released Dec 22, 2021. Pyang 2.6.0 was released Nov 3, 2023. There are no entries for it in the GitHub Advisory Database.

I'd suggest running this by the test team.

Reply:
@singhal-kamalkant can you provide some feedback if these versions are ok to upgrade to from ptf-tests perspective